packages/xpetra/src/BlockedMultiVector/Xpetra_BlockedMultiVector_decl.hpp

```diff
@@ -368,6 +369,7 @@ namespace Xpetra {
       return test;
```

Comment:
If t_dev_um is unmanaged device view of a dual view, this interface is problematic in my opinion. Tpetra with UVM removal does not allow users taking a pointer nor unmanaged view. This is because we want to communicate host and device sync be comparing reference counts. If this pattern is used widely, this interface needs to be fixed.

@kddevin

Reply:
Oh... I am sorry that I misunderstand this code. I don't know why we need this interface returning a dummy view. This is not an actual Tpetra member dual view. So the code itself is fine. If this dummy view is used to get unmanaged view to take Tpetra internal member variable, it is still a bit dangerous as it removes reference counting.

Reply:
I didn't really change that code, I just put it in the ifdef deprecated code and might have cleaned it up some.  It was all over, now it is deprecated,

Reply:
Ah... this interface will be gone. Good~.